src/Commands.pm

```diff
@@ -88,6 +88,7 @@ sub initHandlers {
 	clearlog			=> \&cmdChatLogClear,
 	closeshop			=> \&cmdCloseShop,
 	closebuyshop		=> \&cmdCloseBuyShop,
+	closebuyershop		=> \&cmdCloseBuyerShop,
```

Comment:
cmdCloseBuyShop , cmdCloseBuyerShop 

I think can be used together?

Reply:
merged